### PR TITLE
Send cluster notifications via email

### PR DESCRIPTION
**Pull request by fpob**
### Fixes

- [EXDRHUB-681](https://issues.redhat.com/browse/EXDRHUB-681)

### Checklist

- [x] Related tests were updated
- [x] Run `tox`, no tests failed

### CODE REVIEW

File: config.py [hunk 1]
```diff
@@ -40,7 +40,10 @@ def setup(self, cfg):
 
 logger_class = CustomGunicornLogger
 
-# preload_app = True
+# Must be set to True, otherwise scheduler and messaging thread are started
+# multiple times in each of gunicorn worker.
+preload_app = True
```
Comment (guioliveirabh) [inline config.py:45]:
```suggestion
preload_app = FLASK_ENV != 'development'
```

> The reloader is incompatible with application preloading.

https://docs.gunicorn.org/en/stable/settings.html#reload

I'm not sure if you tried this, but if the incompatibility applies in this case, we may need to remove the 'reload' configuration.

File: .env.defaults [hunk 2]
```diff
@@ -81,3 +81,17 @@ RHUB_SKIP_INIT=
 
 # Default logging level if LOG_CONFIG is not set.
 LOG_LEVEL=info
+
+################################################################################
```
Comment (guioliveirabh) [inline .env.defaults:85]: I think we need to add this to the deployment repositories (rhub-docker, rhub-deployment) on gitlab.

**Review by guioliveirabh — CHANGES_REQUESTED** [review]

File: .env.defaults (hunk 2 above)
Comment (fpob) [reply .env.defaults:85]: These variables are optional and if not defined notifications will be disabled, but yeah, I'll update deployment repos.

File: config.py (hunk 1 above)
Comment (fpob) [reply config.py]: Fixed, I applied your suggestion.

It is a bit tricky, from logs it looks like reloading works, but it doesn't.

**Review by guioliveirabh — APPROVED** [review]

**Pull request merged**